in_toto/in_toto_record.py

```diff
@@ -76,6 +82,7 @@ def main():
   # The subparsers inherit the arguments from the parent parser
   parent_parser = argparse.ArgumentParser(add_help=False)
   subparsers = parser.add_subparsers(dest="command")
+  subparsers.required = True
```

Comment:
Can you comment on why this `required = True` is... required?

Reply:
I hint at it in the related commit message (d74b274177c1ea0dc2ff71f8fceb9138e7d93af2), but I can add a comment too.